README.md

```diff
@@ -1,8 +1,71 @@
-# Jarivs Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Roshan Ramesh . Jarvis Consulting
 
+Experienced recent Electrical Engineering graduate with a knack for creative Problem solving. Skills include delegation, organizational, time-management with prioritizing workflow for optimal throughput. Disciplined, encouraging, strong communicator and a warm extrovert with the ability to bond among introverts with ease. As an Electrical Engineering Graduate aiming to shift into Software(Data), the Jarvis training program promised exactly what was required to hone the data engineering skills through its rigourous industry standard mentoring. Keen to apply learned skills from jarvis in software and IT to improve client, users, employees and administration QOE.
```

Comment:
please do a spelling and grammar check with Grammarly
![image](https://user-images.githubusercontent.com/58951265/99451780-b6426a00-28f0-11eb-9f37-b9f1eb91bb38.png)